cluster/addons/fluentd-gcp/fluentd-gcp-image/Makefile

```diff
@@ -3,14 +3,35 @@
 # the image to DockerHub.
 # Satnam Singh (satnam@google.com)
 
-.PHONY:	build push
+# Procedure for change:
+# 1. Bump the tag number.
+# 2. Create an image for the kubernetes user on Dockerhub and push there: make kbuild kpush
+# 3. Temporarily adjust any config files etc. that reference gcr.io/google_containers/fluentd-gcp:$(TAG)
+#    to use kubernetes/fluentd-gcp:$(TAG) instead.
+# 4. Test etc.
+# 5. Issue PR.
+# 6. When PR is approved make the gcr.io version of the image: make build push
+# 7. Revert the referendes to kubernetes/fluentd-gcp:$(TAG) to gcr.io/google_containers/fluentd-gcp:$(TAG) 
 
+.PHONY:	kbuild kpush
 
-TAG = 1.6
+TAG = 1.7
+
+# Rules for building the test image for deployment to Dockerhub with user kubernetes.
+
+kbuild:
+	docker build -t kubernetes/fluentd-gcp:$(TAG) .
+
+
+kpush:
+	docker push kubernetes/fluentd-gcp:$(TAG)
```

Comment:
No, for `make kpush` we want to push to the kubernetes user on Dockerhub.
For `make push` we want to push to gcr.io
I apologize for the confusion.